### DISCUSSION TITLE
Make a slack alert for people leaving and entering the #co-working-room

### DISCUSSION THREAD

**Discussion by ebanner** [Virtual-Coffee/virtualcoffee.io#1156] [New Ideas]
I saw this co-working room today

![image](https://github.com/Virtual-Coffee/virtualcoffee.io/assets/2068912/99efb054-b791-467d-a36f-5a79aa1762b4)

I actually didn't know there was an active co-working session going on. I think if I could subscribe or opt in to alerts that people are leaving or entering the co-working session would be cool

Might also be cool to make it easier to know whether there is an active co-working session happening right now or not. On discord, it's a little easier to see from this sidebar that people are in the co-working room

<img width="449" alt="image" src="https://github.com/Virtual-Coffee/virtualcoffee.io/assets/2068912/19325226-293c-4c5c-98f3-eae887a19b92">

I don't have a great idea for how to do this, just putting it out there

**Comment by meg-gutshall** [comment]
Hey @ebanner! Slack actually already provides this ability! 👍 

Go to the #co-working-room Slack channel and click on the channel title. (When you hover over the channel title, a popover should appear that reads: 'Get channel details.') You should see something like the screen shot below.

<img width="785" alt="co-working-room-notifications-1" src="https://github.com/Virtual-Coffee/virtualcoffee.io/assets/37842352/fe4d911f-4864-4dfd-a6bb-d12c2227d10a">

Inside that modal, click the button right below the channel title that has a bell icon on it. This is where you can control notifications for that specific channel. If you select 'More notification options' from the dropdown menu, you'll be shown another modal with finer grained settings.

<img width="789" alt="co-working-room-notifications-2" src="https://github.com/Virtual-Coffee/virtualcoffee.io/assets/37842352/76e702dc-2af8-4a8c-82b8-8461082b0d4f">

**Reply by ebanner** [reply]
Ya I'm aware of those settings. The "Get notified about all thread replies in this channel" seems to get the closest to what I want (though it will just be a bunch of coming and leaving messages so I hesitate to enable it)

What I like is how discord has a status indicator that displays whether people are in a voice channel. On slack, I think having that next to the channel name would be what I want. But slack doesn't provide that, so... 🤷‍♂️
